sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.0.0b12 (2025-06-16)
+## 1.0.0b12 (Unreleased)
 
 ### Breaking changes
 
 * These 3 methods on `AIProjectClient` were removed: `.inference.get_chat_completions_client()`,
-`.inference.get_embeddings_client()`, `.inference.get_image_embeddings_client()`. They no longer provide much
-value. To see how you can easily get an authenticated `azure-ai-inference` client for your AI Foundry Project, see
-the updated samples in the folder `samples\inference`. For example,
-[sample_chat_completions_with_azure_ai_inference_client.py](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-projects/samples/inference/sample_chat_completions_with_azure_ai_inference_client.py). Alternatively, us the `.inference.get_azure_openai_client()` method
-to perform chat completions with an Azure OpenAI client.
+`.inference.get_embeddings_client()`, `.inference.get_image_embeddings_client()`.
+To see how you can still easily get an authenticated `azure-ai-inference` client for your AI Foundry Project, see
```

Comment:
consider rewording

For guidance on obtaining an authenticated `azure-ai-inference` client for your AI Foundry Project, refer to the updated samples in the `samples\inference\azure-ai-inference` directory."

Reply:
@jhakulin good suggestion, I updated the text.